### PR TITLE
Correct and update design documentation

### PR DESCRIPTION
**Pull request by SaiSan-WMDE**

### CODE REVIEW

File: docs/src/00-doc/03-create-reusable-components/02_designing_components.stories.mdx [hunk 1]
```diff
@@ -222,23 +237,23 @@ Many of the handover items should already be covered if the design process has b
   - **Visual** specification (sizes, paddings, typographic styles, colors) are fully defined using design tokens ([See 2.2](#22-using-tokens-to-specify-a-new-component)).
   - **States**’ specifications (e.g. default, hover, active, focus, error, disabled) and **interactive behavior** are defined and fully specified. In both cases, the user actions leading to each state should also be indicated.
   - **Responsive** behavior specifications are also outlined <br/>
-<br/>
+    <br />
```
Comment (tzhelyazkova) [inline docs/src/00-doc/03-create-reusable-components/02_designing_components.stories.mdx:240]: Are those space changes on purpose?

File: docs/src/00-doc/03-create-reusable-components/02_designing_components.stories.mdx [hunk 2]
```diff
@@ -222,23 +237,23 @@ Many of the handover items should already be covered if the design process has b
...
 
 👉🏻 The system designer will be in charge of creating and making available, before development, the component-level token files based on the design specifications.
 
 - Basic component documentation (as defined in 1.4):<br/>
   - A distinctive **name** for the component and its relevant typologies
   - A **short description** of the overall purpose of the component<br/>
   - **Accessibility** specifications, such as keyboard navigation, are defined<br/>
-  <br/>
+    <br />
```
Comment (tzhelyazkova) [inline docs/src/00-doc/03-create-reusable-components/02_designing_components.stories.mdx:248]: same here

File: docs/src/00-doc/03-create-reusable-components/02_designing_components.stories.mdx [hunk 3]
```diff
@@ -222,23 +237,23 @@ Many of the handover items should already be covered if the design process has b
...
 - Storybook indications:<br/>
   - Specify where, within the current **Storybook structure**, the component should be classified (new dedicated section or subsection), based on the relationship to its component family (this relationship should also be expressed by the name).
   - Indicate the required Storybook **controls**, if needed: these allow users to modify some properties of a component in real time.<br/>
-  <br/>
+    <br />
```
Comment (tzhelyazkova) [inline docs/src/00-doc/03-create-reusable-components/02_designing_components.stories.mdx:252]: and here

**Review by tzhelyazkova — APPROVED** [review]
Approving because if the spaces are on purpose then this can be merged.

File: docs/src/00-doc/03-create-reusable-components/02_designing_components.stories.mdx (hunk 3 above)
Comment (SaiSan-WMDE) [reply docs/src/00-doc/03-create-reusable-components/02_designing_components.stories.mdx:252]: That was definitely not on purpose 🤔  Fixing

File: docs/src/00-doc/03-create-reusable-components/02_designing_components.stories.mdx [hunk 4]
```diff
@@ -127,20 +130,24 @@ Design tokens are atoms that store the visual foundations of our design system.
...
 
-___Alias tokens___ are abstractions that communicate the token’s intended use. They mostly represent styles that are shared among several components (e.g. "border-color-progressive-active") and take global tokens as values.
+**_Alias tokens_** are abstractions that communicate the token’s intended use. They mostly represent styles that are shared among several components (e.g. "border-color-progressive-active") and take global tokens as values.
 
-___Component-level tokens___ are created by developers and they represent every value associated with the style and interactive properties of a specific component (e.g. button-primary-progressive-border-color-active). They take alias tokens as values.
+**_Component-level tokens_** are created by developers and they represent every value associated with the style and interactive properties of a specific component (e.g. button-primary-progressive-border-color-active). They take alias tokens as values.
 
 <center>
-<img src="./images/token_categ.png" alt="token categorization" width="65%"/>
+  <img src="./images/token_categ.png" alt="token categorization" width="65%" />
```
Comment (SaiSan-WMDE) [inline docs/src/00-doc/03-create-reusable-components/02_designing_components.stories.mdx:140]:
```suggestion
<img src="./images/token_categ.png" alt="token categorization" width="65%" />
```
```suggestion
<img src="./images/token_categ.png" alt="token categorization" width="65%" />
```

File: docs/src/00-doc/03-create-reusable-components/02_designing_components.stories.mdx (hunk 3 above)
Comment (SaiSan-WMDE) [inline docs/src/00-doc/03-create-reusable-components/02_designing_components.stories.mdx:252]:
```suggestion
<br />
```
```suggestion
<br />
```

Comment (SaiSan-WMDE) [inline docs/src/00-doc/03-create-reusable-components/02_designing_components.stories.mdx]:
```suggestion
<br />
```

File: docs/src/00-doc/03-create-reusable-components/02_designing_components.stories.mdx (hunk 2 above)
Comment (SaiSan-WMDE) [inline docs/src/00-doc/03-create-reusable-components/02_designing_components.stories.mdx:248]:
```suggestion
<br />
```

File: docs/src/00-doc/03-create-reusable-components/02_designing_components.stories.mdx (hunk 1 above)
Comment (SaiSan-WMDE) [inline docs/src/00-doc/03-create-reusable-components/02_designing_components.stories.mdx:240]:
```suggestion
<br />
```

**Review by tzhelyazkova — APPROVED** [review]

**Pull request merged**